src/server/api/routers/health-monitoring/openmediavault.ts

```diff
@@ -0,0 +1,118 @@
+import axios from 'axios';
+import Consola from 'consola';
+import { checkIntegrationsType, findAppProperty } from '~/tools/client/app-properties';
+import { getConfig } from '~/tools/config/getConfig';
+import { ConfigAppType } from '~/types/app';
+
+let sessionId: string | null = null;
+let loginToken: string | null = null;
+
+async function makeOpenMediaVaultRPCCall(
+  serviceName: string,
+  method: string,
+  params: Record<string, any>,
+  headers: Record<string, string>,
+  input: { configName: string }
+) {
+  const config = getConfig(input.configName);
+  const app = config.apps.find((app) => checkIntegrationsType(app.integration, ['openmediavault']));
+
+  if (!app) {
+    Consola.error(`App 'openmediavault' not found for configName '${input.configName}'`);
+    return null;
+  }
+
+  const appUrl = new URL(app.url);
+  const response = await axios
+    .post(
+      `${appUrl.origin}/rpc.php`,
+      {
+        service: serviceName,
+        method: method,
+        params: params,
+      },
+      {
+        headers: {
+          'Content-Type': 'application/json',
+          ...headers,
+        },
+      }
+    )
+    .catch((error) => {
+      if (serviceName === 'cputemp') {
+        // ignore cputemp errors; not always supported
```

Comment:
Do not swallow errors. Log as an information (not a warning or error) using Consola logger.